releases/1.4/kubeflow-bundle.yaml

```diff
@@ -0,0 +1,92 @@
+bundle: kubernetes
```

Comment:
Did you use some kind of yaml formatter for this? I'm not entirely a fan of the inconsistent formatting, I think we have to agree on some kind of standard here.

Reply:
Do you mean regarding how applications are single lines like:
```
applications:
  application-name:             { charm: ... }
  longer-application-name:  { charm: ... }
...
```
where the `{}` is aligned?  If so, I'm 90% sure like 6 months ago you argued for this format :smile: with reasoning that it is concise and easy to see which charms are in the bundle.  And it is, but it's also a pain to format.  Do you have a preferred standard?  Typical yaml-by-whitespace feels suitable, although it does make longer, harder to glance at bundle files.

Reply:
I think it was just what was historically done in the `bundle-kubeflow` repo. I personally believe we should just use a formatter so that we don't have to have these conversations in the future.

[This one](https://jsonformatter.org/yaml-formatter) seems to use yaml-by-whitespace, but not sure if a online formatter is what we want.